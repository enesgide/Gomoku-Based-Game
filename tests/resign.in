place A2
history
resign