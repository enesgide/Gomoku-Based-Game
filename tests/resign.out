A2
Black wins!
A2
Thank you for playing!
